README.md

```diff
@@ -100,7 +100,7 @@ you should see somewhere around 100% performance increase.
 
 ## Commitment to Quality
 During package development I try as best as possible to embrace good design and
-development practices to try to ensure that this package is as good as it can
+development practices trying to ensure that this package is as good as it can
```

Comment:
perhaps a comma is called for here instead: `good design and development practices, to help ensure`